### PR TITLE
Some edits and de-duplication

### PR DESCRIPTION
**Pull request by apragsdale**
Addressing some comments and de-duplicating some of the sections that overlapped with the Appendix. A work in progress.

### CODE REVIEW

**Comment by molpopgen** [comment]
Looks good to me, and it removes a lot, which is good at this point.

**Review by grahamgower — APPROVED** [review]
LGTM

File: paper.tex [hunk 1]
```diff
@@ -627,8 +588,8 @@ \subsection{Selfing and cloning}
 More details of the mathematical properties of selfing and cloning rates
 in a coalescent context can be found in \citet{hartfield2016facsexcoal}.
```
Comment (MattHartfield) [inline paper.tex]: Also cite Nordborg and Donnelly 1997 here?

File: paper.tex [hunk 2]
```diff
@@ -881,6 +817,7 @@ \section{Extended data and figures}
         form. (C) A visual representation of the model using \texttt{demesdraw}.
         The YAML descriptions in (A) and (B) correspond exactly to a JSON description,
         but are much more human-readable.
+        \aprcomment{Need figure panel labels.}
```
Comment (MattHartfield) [inline paper.tex:823]: Do we also need to delete these lines?

        \begin{tcolorbox}
            \includegraphics[width=\textwidth]{fig/IM}
        \end{tcolorbox}
        
When I try and compile the .tex file on my machine then it throws an error as this figure is missing; I take it that it's been superseded by 'models/IM.yaml'?

**Review by MattHartfield — COMMENTED** [review]
Looks good; I've a few minor comments.

File: paper.tex (hunk 2 above)
Comment (grahamgower) [reply paper.tex:823]: That's strange, as it should be generated by the Makefile if it is missing. Can you paste the full error message when you do `make`?

Comment (MattHartfield) [reply paper.tex:823]: Ah, I get it now, I was trying to create the PDF using latex rather than the makefile. Now it works fine!

File: paper.tex [hunk 3]
```diff
@@ -302,47 +303,34 @@ \subsection*{Parsers}
...
-\krtcomment{Here, we should say something about other models that people may wish to implement within the demes
-framework.}
-
-\krtcomment{
-    I think we want a section in the Discussion outlining some obvious things
-    that people can do w/demes that are sitting "just outside" the strict
-    specification: hard selection, etc..
-}
+However, we note that Demes may be used in downstream applications that
+do not conform to the specification. This might include interpreting
```
Comment (jeromekelleher) [inline paper.tex]: I'd avoid saying "not conforming" here, as that sounds like people are doing things we don't want them to. Really this is a designed-in flexibilty. Maybe something like 

> It is important to note, however, that Demes may be used in applications that include additional population genetic processes outside of what is explicitly modelled in the specification, such as X Y. We can see the Demes specification as providing a basic model that can be elaborated on where necessary.

File: paper.tex [hunk 4]
```diff
@@ -748,39 +709,14 @@ \section{Scope of the specification}
...
-%\item Simulating from some arbitrary joint distribution that was inferred
-%      previously. E.g. using an ABC posterior distribution of models to
-%      train a neural network.
-%\item Extending someone else's static model to reinfer parameters,
-%      or add new parameters.
-%\end{enumerate}
-%}
+The Demes specification is designed to describe demographic
+models defined by a fixed set of model parameters.
+As described in the main text, the Demes specification does not include
```
Comment (jeromekelleher) [inline paper.tex]:
```suggestion
As described in the main text, it does not include
```

**Review by jeromekelleher — APPROVED** [review]
LGTM, minor suggestions

File: paper.tex (hunk 3 above)
Comment (apragsdale) [reply paper.tex]: Yes, that's much better. Thanks!

**Pull request merged**